besspin/cyberPhys/ui/hacker-kiosk/src/background.js

```diff
@@ -25,6 +27,31 @@ ipcMain.on('can-poll', (event) => {
   }
 })
 
+// ZMQ Network
+let zmq_sock = zmq.socket("pair");
+let zmq_address = "tcp://127.0.0.1:3333";
+zmq_sock.connect(zmq_address);
```

Comment:
Do you want to make this compatible with the Python interface in this PR or in a subsequent PR?